suiteData.js

```diff
@@ -32,6 +38,15 @@ class SuiteData {
                 suiteData = await this.getSuiteData(request.query);
                 break;
             case "POST":
+                let tests = request.body?.tests;
+                if (_.isString(tests)) {
+                    const testsDir = path.join(config.testsDir, sanitize(tests));
+                    if (await exists(testsDir)) {
+                        request.body.tests = (await listDir(testsDir)).filter(f => path.extname(f) === '.transcript').map(f => ({path: path.join(testsDir, f)}));
```

Comment:
better not to change the body on the request object itself. it can get confusing since this is shared by all the routes and middlewares.
construct the data here instead:
```
suiteData = new SuiteData((await ...).filter(...).map(...), request.query);
```

Reply:
fixed